ansible/install_keystone.yaml

```diff
@@ -55,3 +55,16 @@
     environment:
       PATH: "{{ oc_env_path }}"
       KUBECONFIG: "{{ kubeconfig }}"
+
+  # This workaround can be removed when
+  # https://github.com/openstack-k8s-operators/keystone-operator/issues/30 is
+  # resolved.
+  - name: Wait for keystone bootstrap
+    shell: |
+      openstack user show admin
+    environment:
+      <<: *oc_env
+    register: keystone_bootstrap
+    until: keystone_bootstrap is not failed
+    retries: "{{ (default_timeout / 5)|int }}"
+    delay: 5
```

Comment:
Instead of this you can actually look for the bootstrap hash directly on the keystone CR. It is under the 'Status' and should ensure that bootstrap is finished.